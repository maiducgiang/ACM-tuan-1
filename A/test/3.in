5
A H
H A
F E
E F
Y C