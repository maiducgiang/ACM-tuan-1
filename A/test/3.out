A => B => C => D => H
H => D => C => B => A
F => B => C => D => E
E => D => C => B => F
Y => H => D => C
